iot-hub/Samples/service/AzureSasCredentialAuthenticationSample/Program.cs

```diff
@@ -58,7 +58,7 @@ public static async Task Main(string[] args)
             await sample.RunSampleAsync(serviceClient, parameters.DeviceId);
         }
 
-        private static string GenerateSasToken(string resourceUri, string sharedAccessKey, DateTime expiresOn, string policyName = default )
+        private static string GenerateSasToken(string resourceUri, string sharedAccessKey, string policyName, DateTime expiresOn)
```

Comment:
policy name is required for sas token generation, throws unauth exception otherwise